pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,491 @@
+
+## Managing Docker Service on Windows using PowerShell
+
+On Windows, Docker typically runs as a service via Docker Desktop. While Docker Desktop provides a user-friendly interface for managing Docker, there are instances where you might want to control the Docker service directly from the command line using PowerShell.
+
+Here's how you can manage the Docker service on Windows using PowerShell:
+
+### Open PowerShell as an Administrator
+
+- Right-click the Start button and select "Windows PowerShell (Admin)" or search for "PowerShell" in the start menu, right-click "Windows PowerShell", and select "Run as administrator".
+
+### Start the Docker Service
+
+If Docker isn't running and you wish to start it, use the following command:
+
+```powershell
+net start com.docker.service
+```
+
+You should receive a message indicating that the Docker service has started successfully.
+
+### Stop the Docker Service
+
+To stop the Docker service, use the following command:
+
+```powershell
+net stop com.docker.service
+```
+
+You should receive a message indicating that the Docker service has stopped.
+
+### Check the Status of the Docker Service
+
+To verify the status of the Docker service, whether it's running or stopped, use:
+
+```powershell
+Get-Service com.docker.service
+```
+
+This command will display the status of the Docker service.
+
+</Callout>
+
+## Work with Docker
+
+If you are new to Docker, this guide will help you get a grasp of Docker and
+make it easier to accomplish tasks within Memgraph. After installing Docker, all
+commands are run from the command-line tool of your preference.
+
+### Download the Memgraph Docker image
+
+<Callout type="info">
+When you pull a Memgraph Docker image, think of it as obtaining the blueprint
+for Memgraph, complete with all its core features. This image is your starting
+point. From it, you can launch a container, which acts as a live instance of
+Memgraph. All your interactions, queries, and transactions with Memgraph occur
+within this container. This ensures a consistent and isolated setting for all
+your database tasks. From a single image, you can spawn multiple
+containers. Each container functions as a distinct Memgraph instance.
+</Callout>
+
+
+Images are downloaded using the `docker pull` command followed by the name of
+the Docker image. We encourage you to use Memgraph Platform as it includes
+everything you might need while making the best of Memgraph.
+
+To download the latest Memgraph Platform image, run:
+
+```
+docker pull memgraph/memgraph-platform
+```
+
+All Memgraph Docker images are availble at [Memgraph's Docker
+Hub](https://hub.docker.com/u/memgraph). There you will find exact Docker image
+tags you can downalod and use to install Memgraph with. 
+
+### Architecture of Docker container running Memgraph
+
+The picture below shows the architecture of the Memgraph Docker ecosystem.
+
+![](/pages/getting-started/install-memgraph/docker/docker-architecture.png)
```

Comment:
As this is now separate from the main docker installation, maybe here we could list all the images Memgraph has, show this "blueprint" what you get with each image

Also please look at what language i used to described the images and when to use which